src/common/types.cpp

```diff
@@ -30,12 +30,33 @@
 #include <sstream>
 #include <sys/socket.h>
 
+#ifdef __APPLE__
+#include <libkern/OSByteOrder.h>
+#define htobe16(x) OSSwapHostToBigInt16(x)
```

Comment:
Avoid defining macros. Functions should be preferred here.

Reply:
Are there any special reasons to avoid defining macros?

Reply:
Macros don't have namespace, so they are likely to conflict with other names. Macros also have many well known pitfalls. And this is also included in [the style guide](https://google.github.io/styleguide/cppguide.html#Preprocessor_Macros).